src/yacs-web/src/components/Header.vue

```diff
@@ -1,7 +1,7 @@
 <template>
     <b-navbar type="light" variant="light">
         <b-navbar-brand class = "logo" href="#">YACS</b-navbar-brand>
-
+        
 		<b-form-select label= "semester" class="semesters" v-model="selectedSemester" :options="semesterOptions">
```

Comment:
if you add a `text-field="semester"` and `value-field="semester"` to the `b-form-select.semesters`, then in `created ()` you can just push `data` returned by `getSemesters` instead of creating a new array.

Reply:
I am just mimicking all of the other similar methods.